### PR TITLE
Red 3.4.10 - Changelog

### PR DESCRIPTION
**Pull request by Jackenmen**
There will eventually be a changelog here...

Link to the rendered changelog: https://red-discordbot--5077.org.readthedocs.build/en/5077/changelog_3_4_0.html#redbot-3-4-10-2021-05-28

### CODE REVIEW

**Comment by Jackenmen** [comment]
Should be ready for review, might need an update if the remaining PRs in the milestone get merged.

File: docs/changelog_3_4_0.rst [hunk 1]
```diff
@@ -1,5 +1,142 @@
...
+Redbot 3.4.10 (2021-05-28)
+==========================
+
+| Thanks to all these amazing people that contributed to this release:
+| :ghuser:`aikaterna`, :ghuser:`aleclol`, :ghuser:`benno1237`, :ghuser:`bobloy`, :ghuser:`BoyDownTown`, :ghuser:`Danstr5544`, :ghuser:`DeltaXWizard`, :ghuser:`Drapersniper`, :ghuser:`Fabian-Evolved`, :ghuser:`fixator10`, :ghuser:`Flame442`, :ghuser:`flaree`, :ghuser:`jack1142`, :ghuser:`Kowlin`, :ghuser:`Kreusada`, :ghuser:`Lifeismana`, :ghuser:`Obi-Wan3`, :ghuser:`OofChair`, :ghuser:`palmtree5`, :ghuser:`plofts`, :ghuser:`PredaaA`, :ghuser:`Predeactor`, :ghuser:`TrustyJAID`, :ghuser:`Vexed01`
+
+Read before updating
+--------------------
+
+1. PM2 process manager is no longer supported is as it is not a viable solution due to certain parts of its behavior.
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
1. PM2 process manager is no longer supported as it is not a viable solution due to certain parts of its behavior.
```

File: docs/changelog_3_4_0.rst [hunk 2]
```diff
@@ -1,5 +1,142 @@
...
+
+
+End-user changelog
+------------------
+
+Core Bot
+********
+
+- Fixed terminal colors on Windows (:issue:`5063`)
+- Fixed ``--rich-traceback-extra-lines`` flag (:issue:`5028`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- Fixed the ``--rich-traceback-extra-lines`` flag (:issue:`5028`)
```

File: docs/changelog_3_4_0.rst [hunk 3]
```diff
@@ -1,5 +1,142 @@
...
+- Added missing information about the ``showaliases`` setting in ``[p]helpset showsettings`` command (:issue:`4971`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- Added missing information about the ``showaliases`` setting in ``[p]helpset showsettings`` (:issue:`4971`)
```

File: docs/changelog_3_4_0.rst [hunk 4]
```diff
@@ -1,5 +1,142 @@
...
+- Help command no longer errors when it doesn't have permission to read message history and menus are enabled (:issue:`4959`, :issue:`5030`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- The help command no longer errors when it doesn't have permission to read message history and menus are enabled (:issue:`4959`, :issue:`5030`)
```

File: docs/changelog_3_4_0.rst [hunk 5]
```diff
@@ -1,5 +1,142 @@
...
+- Fixed a bug in ``[p]embedset user`` that made it impossible to reset the user's embed setting (:issue:`4962`)
+- ``[p]embedset command`` command and its subcommands now properly check whether any of the passed command's parents require Embed Links permission (:issue:`4962`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- ``[p]embedset command`` and its subcommands now properly check whether any of the passed command's parents require Embed Links permission (:issue:`4962`)
```

File: docs/changelog_3_4_0.rst [hunk 6]
```diff
@@ -1,5 +1,142 @@
...
+- Fixed an issue with Red reloading unrelated modules when using ``[p]load`` and ``[p]reload`` (:issue:`4956`, :issue:`4958`)
+
+Admin
+*****
+
+- The cog will now log when it leaves the guild due to the serverlock (:issue:`5008`, :issue:`5073`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- The cog will now log when it leaves a guild due to the serverlock (:issue:`5008`, :issue:`5073`)
```

File: docs/changelog_3_4_0.rst [hunk 7]
```diff
@@ -1,5 +1,142 @@
...
+- Fixed an issue that made it possible to remove Aikaterna's curated tracks playlist (:issue:`5018`)
+
+Audio
+*****
+
+- Fixed auto-resume of auto play after Lavalink restart (:issue:`5051`)
+- ``[p]audiostats`` command can now only be used by bot owners (:issue:`5017`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- The ``[p]audiostats`` command can now only be used by bot owners (:issue:`5017`)
```

File: docs/changelog_3_4_0.rst [hunk 8]
```diff
@@ -1,5 +1,142 @@
...
+- Fixed an error with ``[p]audiostats`` caused by players not always having their connection time stored (:issue:`5046`)
+- Fixed track resuming in a certain edge case (:issue:`4996`)
+- Fixed an error in ``[p]audioset restart`` command (:issue:`4987`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- Fixed an error in ``[p]audioset restart`` (:issue:`4987`)
```

File: docs/changelog_3_4_0.rst [hunk 9]
```diff
@@ -1,5 +1,142 @@
...
+Downloader
+**********
+
+- ``[p]repo remove`` can now remove multiple repos at the same time (:issue:`4765`, :issue:`5082`)
+- ``[p]cog install`` now properly shows the repo name rather than ``{repo.name}`` (:issue:`4954`)
+
+Mod
+***
+
+- ``[p]mute`` command no longer errors on muting a bot user if ``senddm`` option is enabled (:issue:`5071`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- ``[p]mute`` no longer errors on muting a bot user if the ``senddm`` option is enabled (:issue:`5071`)
```

File: docs/changelog_3_4_0.rst [hunk 10]
```diff
@@ -1,5 +1,142 @@
...
+
+Miscellaneous
+-------------
+
+- Clarified that ``[p]cleanup`` commands only delete the messages from the current channel (:issue:`5070`)
+- Updated Python version in ``pyenv`` and Windows instructions (:issue:`5025`)
+- Added information on how to set the bot not to start on boot anymore to auto-restart docs (:issue:`5020`)
+- Improved logging in Audio cog (:issue:`5044`)
+- Improved logging of API errors in Streams cog (:issue:`4995`)
+- ``[p]urban`` command from General cog will now use the default embed color of the bot (:issue:`5014`)
```
Comment (Flame442) [inline docs/changelog_3_4_0.rst]:
```suggestion
- The command ``[p]urban`` from the General cog will now use the default embed color of the bot (:issue:`5014`)
```

**Review by Flame442 — CHANGES_REQUESTED** [review]

**Review by Flame442 — APPROVED** [review]
🎉

**Pull request merged**